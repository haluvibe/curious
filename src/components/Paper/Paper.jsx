import React from "react";
import { withStyles } from "material-ui/styles";
import Paper from "material-ui/Paper";

const styles = theme => ({
  paper: {
    padding: theme.spacing.unit * 2,
    textAlign: "center",
    color: theme.palette.text.secondary
  }
});

const Cell = ({ classes, children }) => 
  <Paper className={classes.paper}>{children}</Paper>
;

Cell.defaultProps = {
  xs: 12
};

export default withStyles(styles)(Cell);
