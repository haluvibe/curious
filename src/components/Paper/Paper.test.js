import React from "react";
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import Paper from "./Paper";

test("Paper component should render as expected", () => {
  const component = shallow(<Paper />),
    tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
