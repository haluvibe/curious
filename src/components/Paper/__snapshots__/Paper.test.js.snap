// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper component should render as expected 1`] = `
<Cell
  classes={
    Object {
      "paper": "Cell-paper-1",
    }
  }
  xs={12}
/>
`;
